bindings/java/c/evmc-vm.c

```diff
@@ -97,7 +97,8 @@ JNIEXPORT jobject JNICALL Java_org_ethereum_evmc_EvmcVm_execute(JNIEnv* jenv,
                                                                 jobject jcontext,
                                                                 jint jrev,
                                                                 jobject jmsg,
-                                                                jobject jcode)
+                                                                jobject jcode,
+                                                                jobject jinputdata)
```

Comment:
```suggestion
                                                                jobject jinput)
```

Would name it like this if there are no symbol clashes.